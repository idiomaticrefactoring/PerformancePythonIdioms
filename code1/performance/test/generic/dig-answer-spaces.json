[{"id":26965,"opcode":"QUERY","status":"NXDOMAIN","flags":["qr","rd","ra"],"query_num":1,"answer_num":0,"authority_num":1,"additional_num":1,"opt_pseudosection":{"edns":{"version":0,"flags":[],"udp":4096}},"question":{"name":"x.y.z.w.bl.spamcop.net.","class":"IN","type":"TXT"},"authority":[{"name":"bl.spamcop.net.","class":"IN","type":"SOA","ttl":0,"data":"bl.spamcop.net. hostmaster.admin.spamcop.net. 1612535191 3600 1800 3600 0"}],"answer":[{"name":"x.y.z.w.bl.spamcop.net.","class":"IN","type":"TXT","ttl":2100,"data":"Blocked - see https://www.spamcop.net/bl.shtml?w.z.y.x"}],"query_time":297,"server":"192.168.1.254#53(192.168.1.254)","when":"Fri Feb 05 06:28:58 PST 2021","rcvd":104,"when_epoch":1612535338,"when_epoch_utc":null}]
