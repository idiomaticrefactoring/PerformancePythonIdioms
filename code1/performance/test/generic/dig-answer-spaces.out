
; <<>> DiG 9.10.6 <<>> TXT x.y.z.w.bl.spamcop.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 26965
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;x.y.z.w.bl.spamcop.net.        IN  TXT

;; AUTHORITY SECTION:
bl.spamcop.net.     0   IN  SOA bl.spamcop.net. hostmaster.admin.spamcop.net. 1612535191 3600 1800 3600 0

;; ANSWER SECTION:
x.y.z.w.bl.spamcop.net. 2100 IN TXT "Blocked - see https://www.spamcop.net/bl.shtml?w.z.y.x"

;; Query time: 297 msec
;; SERVER: 192.168.1.254#53(192.168.1.254)
;; WHEN: Fri Feb 05 06:28:58 PST 2021
;; MSG SIZE  rcvd: 104

